### PR TITLE
Add UITest docs

### PR DESCRIPTION
**Pull request by jalkire**
#### Describe the change
Added UITest docs

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: src/UITests/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## UI Tests
+* UI tests reside in the `UITests` project.
+* [WinAppDriver](https://github.com/Microsoft/WinAppDriver) is used for navigation and [Axe.Windows](https://github.com/microsoft/axe-windows) is used for accessibility scanning.
+* Test classes inherit from `AIWinSession`.
+* Each test class covers an AI-Win usage scenario. For example, the `LoadTestFile` test covers the scenario of loading a test file and inspecting indiviudal test results. Each test class has at least the following methods:
```
Comment (karanbirsingh) [inline src/UITests/README.md]: spelling of individual
```suggestion
* Each test class covers an AI-Win usage scenario. For example, the `LoadTestFile` test covers the scenario of loading a test file and inspecting individual test results. Each test class has at least the following methods:
```

File: src/UITests/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+## UI Tests
```
Comment (karanbirsingh) [inline src/UITests/README.md]: I think we should add info about how to run the test (e.g. running winappdriver)

File: src/UITests/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* UI tests reside in the `UITests` project.
```
Comment (karanbirsingh) [inline src/UITests/README.md]: Can we also add brief information about possible artifacts in ADO?
- a11ytest file
- events file
- screen recording

File: src/UITests/README.md (hunk 2 above)
Comment (jalkire) [reply src/UITests/README.md]: Info added

File: src/UITests/README.md (hunk 3 above)
Comment (jalkire) [reply src/UITests/README.md]: Info added

**Pull request merged**